Project2015To2017Tests/ProjectPropertiesReadTest.cs

```diff
@@ -642,142 +552,16 @@ public async Task ReadsUnknownConfigurations()
 			Assert.IsNull(project.AdditionalPropertyGroups[3].Attribute("Condition"));
 
 			var childrenDebug = project.AdditionalPropertyGroups[0].Elements().ToImmutableArray();
-			Assert.AreEqual(0, childrenDebug.Length);
+			Assert.AreEqual(2, childrenDebug.Length);
```

Comment:
Project simplification is executed no more, test is duplicated in `PropertySimplificationTransformation` to check simplification.